scripts/validate-version.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Let's also use `#!/usr/bin/env bash` like we are doing in the other script, just to keep things consistent?